packages/clamav-rest-server/__tests__/fixtures/appeal.js

```diff
@@ -0,0 +1,37 @@
+const appealFromAppellant = {
```

Comment:
Is this file used in the unit tests?

Reply:
No it isn't actually, looks like hangover from a service that I copied.